rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/280_geohash_grid.yml

```diff
@@ -0,0 +1,68 @@
+setup:
+  - do:
+      indices.create:
+          index: test_1
+          body:
+            settings:
+              number_of_replicas: 0
+            mappings:
+              _doc:
+                properties:
+                  location:
+                     type: geo_point
+
+---
+"Basic test":
+  - do:
+      bulk:
+        refresh: true
+        body:
+          - index:
+              _index: test_1
+              _type:  _doc
```

Comment:
ah. mental note. need to remove mention of type as is done in https://github.com/elastic/elasticsearch/pull/37611